packages/manager/cypress/e2e/core/firewalls/update-firewall.spec.ts

```diff
@@ -396,4 +396,62 @@ describe('update firewall', () => {
         });
     });
   });
+
+  /*
+   * - Confirms that firewall's label can be updated on landing page'.
+   */
+  it("updates a firewall's label", () => {
+    const region = chooseRegion();
+
+    const linodeRequest = createLinodeRequestFactory.build({
+      label: randomLabel(),
+      region: region.id,
+      root_pass: randomString(16),
+    });
+
+    const firewallRequest = firewallFactory.build({
+      label: randomLabel(),
+      rules: {
+        inbound: [],
+        outbound: [],
+      },
+    });
+
+    const newFirewallLabel = randomLabel();
+
+    cy.defer(
+      createLinodeAndFirewall(linodeRequest, firewallRequest),
+      'creating Linode and firewall'
+    ).then(([_linode, firewall]) => {
+      cy.visitWithLogin('/firewalls');
+
+      // Confirm that firewall is listed on landing page with expected configuration.
+      cy.findByText(firewall.label)
+        .closest('tr')
+        .within(() => {
+          cy.findByText(firewall.label).should('be.visible');
+          cy.findByText('Enabled').should('be.visible');
+          cy.findByText('No rules').should('be.visible');
+          cy.findByText('None assigned').should('be.visible');
+        });
+
+      cy.visitWithLogin(`/firewalls/${firewall.id}`);
+
+      cy.get(`[aria-label="Edit ${firewall.label}"]`).click();
+      cy.get(`[id="edit-${firewall.label}-label"]`)
+        .click()
+        .clear()
+        .type(`${newFirewallLabel}{enter}`);
+
+      cy.reload();
+
+      // Confirm firewall label is updated on details page.
+      cy.findByText(newFirewallLabel).should('be.visible');
+
+      cy.visitWithLogin('/firewalls');
+
+      // Confirm firewall label is updated on landing page.
+      cy.findByText(newFirewallLabel).closest('tr').should('be.visible');
```

Comment:
```suggestion
      // Confirm Firewall label updates in breadcrumbs.
      ui.entityHeader
        .find()
        .within(() => {
          cy.findByText(newFirewallLabel).should('be.visible');
          cy.findByText('firewalls').click();
        });

      // Confirm firewall label is updated on landing page without refresh.
      cy.findByText(newFirewallLabel).should('be.visible');

      // Confirm firewall label is updated on landing page after refresh.
      cy.reload();
      cy.findByText(newFirewallLabel).should('be.visible');
```
Just a really minor suggestion to _slightly_ improve the coverage by confirming that the label update takes effect without a page refresh (i.e. the React Query cache was successfully updated) and then refreshes the page and confirms the label again (confirming that the label has been updated successfully on the backend, too).